src/patternfly/components/Page/page.scss

```diff
@@ -126,6 +126,23 @@
     --pf-c-page__main-nav--PaddingLeft: var(--pf-c-page__main-nav--md--PaddingLeft);
   }
 
+  // Main section breadcrumb
+  --pf-c-page__main-breadcrumb--BackgroundColor: var(--pf-global--BackgroundColor--light-100);
+  --pf-c-page__main-breadcrumb--PaddingTop: var(--pf-global--spacer--md);
+  --pf-c-page__main-breadcrumb--PaddingRight: var(--pf-global--spacer--md);
+  --pf-c-page__main-breadcrumb--PaddingBottom: 0;
+  --pf-c-page__main-breadcrumb--PaddingLeft: var(--pf-global--spacer--md);
+  --pf-c-page__main-breadcrumb--md--PaddingTop: var(--pf-global--spacer--xl);
+  --pf-c-page__main-breadcrumb--md--PaddingRight: var(--pf-global--spacer--xl);
+  --pf-c-page__main-breadcrumb--md--PaddingLeft: var(--pf-global--spacer--xl);
+  --pf-c-page__main-nav--main-breadcrumb--PaddingTop: 0;
```

Comment:
I'm not sure we've had to write a var for this selector between two different elements, but I think it works.